bindings/python/test/environment/objects/celestial/test_earth.py

```diff
@@ -0,0 +1,201 @@
+
+# @project        Open Space Toolkit ▸ Physics
+# @file           bindings/python/test/objects/celestial/test_earth.py
+# @author         Antoine Paletta <antoine.paletta@loftorbital.com>
+# @license        Apache License 2.0
+
+################################################################################################################################################################
+
+import pytest
+
+from ostk.physics.environment.objects.celestial_bodies import Earth
+
+from ostk.physics.environment.objects.celestial_bodies.earth.models import EGM2008
+from ostk.physics.environment.objects.celestial_bodies.earth.models import EGM96
+from ostk.physics.environment.objects.celestial_bodies.earth.models import WGS84_EGM96
+from ostk.physics.environment.objects.celestial_bodies.earth.models import EGM84
+from ostk.physics.environment.objects.celestial_bodies.earth.models import WGS84
+from ostk.physics.environment.objects.celestial_bodies.earth.models import Spherical
+
+################################################################################################################################################################
+
+def test_earth_static_default_constructor ():
+
+    earth: Earth = Earth.default()
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+################################################################################################################################################################
+
+def test_earth_static_EGM2008_constructor ():
+
+    earth: Earth = Earth.EGM2008()
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+    earth: Earth = Earth.EGM2008(2190,2160)
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+################################################################################################################################################################
+
+def test_earth_static_EGM96_constructor ():
+
+    earth: Earth = Earth.EGM96()
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+    earth: Earth = Earth.EGM96(360,360)
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+################################################################################################################################################################
+
+def test_earth_static_EGM84_constructor ():
+
+    earth: Earth = Earth.EGM84()
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+    earth: Earth = Earth.EGM84(180,180)
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+################################################################################################################################################################
+
+def test_earth_static_WGS84_EGM96_constructor ():
+
+    earth: Earth = Earth.WGS84_EGM96()
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+    earth: Earth = Earth.WGS84_EGM96(180,180)
```

Comment:
```suggestion
    earth: Earth = Earth.WGS84_EGM96(180, 180)
```
and `earth` naming should not be re-used, give it a different name